### PR TITLE
i18n(fr): Add `cms/strapi.mdx` in French language

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description (required)
Add `cms/strapi.mdx` in French language

#### Related issues & labels (optional)

No related issues. /

### CODE REVIEW

**Review by McFlyPartages — COMMENTED** [review]
Quelques suggestions sans obligations.

Sinon super boulot LGTM ;)

File: src/content/docs/fr/guides/cms/strapi.mdx [hunk 1]
```diff
@@ -0,0 +1,425 @@
...
+type: cms
+service: Strapi
+stub: false
+i18nReady: true
+---
+
+import FileTree from '~/components/FileTree.astro';
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro';
+
+[Strapi](https://strapi.io/) est open-source, personnalisable et un CMS headless.
```
[the remark this answers is not in the document]
Comment (thomasbnt) [reply src/content/docs/fr/guides/cms/strapi.mdx:13]: > ```suggestion
> [Strapi](https://strapi.io/) est un CMS Headless open-source et personnalisable.
> ```

Ah merci, je ne savais pas comment le formuler.

**Comment by McFlyPartages** [comment]
LGTM

**Review by sarah11918 — APPROVED** [review]
Merci! 🙌

**Pull request merged**